tools/mo/openvino/tools/mo/utils/ir_reader/extenders/if_extender.py

```diff
@@ -20,7 +20,12 @@ def set_input_output_id(subgraph, input_port_map, output_port_map):
                     node['input_id'] = if_input_mapping_elem['external_port_id']
             for if_out_mapping_elem in output_port_map:
                 if node_id == if_out_mapping_elem['internal_layer_id']:
-                    node['output_id'] = if_out_mapping_elem['external_port_id']
+                    # If external_point ID is counted with inputs 
+                    if if_out_mapping_elem['external_port_id'] > num_of_out_ports:
+                        node['output_id'] = if_out_mapping_elem['external_port_id'] - num_of_in_ports
+                    # If external_point ID is counted from 0 
+                    else:
+                        node['output_id'] = if_out_mapping_elem['external_port_id']
```

Comment:
Let me double-check this one before merge because it is sensitive change.
What if some ports are not connected in `If` will you cont them?
Do we need to make changes for `Loop` operation?
Thanks!